roles/bootstrap/tasks/main.yml

```diff
@@ -64,12 +43,15 @@
 - name: Set hostname
   ansible.builtin.hostname:
     name: "{{ inventory_hostname }}"
+    use: debian
```

Comment:
sure you wanna hardcode this? we use said role for managing our apple nodes  as well

Reply:
no we don't :D

Reply:
ah cool